src/status_im/contexts/profile/edit/list_items.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im.contexts.profile.edit.list-items
+  (:require [quo.foundations.colors :as colors]
+            [status-im.common.not-implemented :as not-implemented]
+            [status-im.contexts.profile.edit.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn items
+  []
+  (let [profile             (rf/sub [:profile/profile-with-image])
+        customization-color (rf/sub [:profile/customization-color])
+        full-name           (profile.utils/displayed-name profile)]
+    [{:label (i18n/label :t/profile)
+      :items [{:title           (i18n/label :t/name)
+               :on-press        #(rf/dispatch [:open-modal :edit-name])
+               :blur?           true
+               :label           :text
+               :label-props     full-name
+               :action          :arrow
+               :container-style style/item-container}
+              {:title           (i18n/label :t/bio)
+               :on-press        not-implemented/alert
+               :blur?           true
+               :action          :arrow
+               :container-style style/item-container}
+              {:title           (i18n/label :t/accent-colour)
+               :on-press        not-implemented/alert
+               :label           :color
+               :label-props     (colors/resolve-color customization-color :dark)
```

Comment:
why are we hard coding theme here?
it's better to wrap these pages in a theme provider context with `:dark` as the value and pass the theme directly.